src/components/upload-modal/__tests__/UploadModal.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { fireEvent, render, screen } from '@testing-library/react'
+import { describe, it, expect, vi } from 'vitest'
+import UploadModal from '..'
+import { Button } from '../../ui/button'
+
+describe('UploadModal', () => {
```

Comment:
Do we need the describe here? It is one per file, has the same name as the file and doesn't have any side effects / before / after hooks

Reply:
I like that it shows what we're testing in a normal text, not file path, but sure, isn't necessary here, removed.